lab-cortney/gulpfile.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+const gulp = require('gulp');
+const mocha = require('gulp-mocha');
+const eslint = require('gulp-eslint');
+
+gulp.task('default', function() {
+  return gulp.src(__dirname + '/test/greet-test.js')
+    .pipe(mocha());
+});
+
+gulp.task('lint', function() {
```

Comment:
This looks good, you can make your file path to watch simpler, "**/*.js", that will run the linter on every .js file in your directory.  Since it's looking at everything you'll want to also make sure to ignore the appropriate files (everything in node_modules).  So in the end you'll want it to look like this:
['**/*.js', '!node_modules/**']